sys/arm64/include/armreg.h

```diff
@@ -64,6 +64,16 @@
 #define	WRITE_SPECIALREG(reg, _val)					\
 	__asm __volatile("msr	" __STRING(reg) ", %0" : : "r"((uint64_t)_val))
 
+#if __has_feature(capabilities)
+#define	READ_SPECIALREG_CAP(reg)						\
+({	uintcap_t _val;							\
+	__asm __volatile("mrs	%0, " __STRING(reg) : "=&C" (_val));	\
```

Comment:
Doesn't need to be volatile but I guess it's consistent with the non-_CAP version upstream